admin/includes/admin_html_head.php

```diff
@@ -22,12 +22,13 @@
     <link rel="stylesheet" href="<?php echo $value; ?>">
     <link rel="stylesheet" href="includes/fontawesome/css/solid.min.css">
     <link rel="stylesheet" href="includes/fontawesome/css/regular.min.css">
-    <?php if (!empty($fontawesome_v4_shim_required) && file_exists($value = 'includes/fontawesome/css/v4-shims.min.css')) { ?>
+    <?php if ((empty($disableFontAwesomeV4Compatibility)) &&
+        file_exists($value = 'includes/fontawesome/css/v4-shims.min.css')) { ?>
```

Comment:
Missed this before; the file name should be `DIR_WS_INCLUDES . 'fontawesome/css/v4-shims.min.css'`

Reply:
I've pushed this change, it extended well beyond this line, I guess all those old lines also need updating.  Didn't touch the plugin includes stuff as presumably that directory structure doesn't use DIR_WS_INCLUDES in the same way.